api/v1/swagger/paths/erm_counter_files.yaml

```diff
@@ -0,0 +1,282 @@
+      200:
+        description: A list of counter_files
+        schema:
+          items:
+            $ref: "../swagger.yaml#/definitions/erm_counter_file"
+          type: array
+      400:
+        description: Bad request
+        schema:
+          $ref: "../swagger.yaml#/definitions/error"
+      403:
+        description: Access forbidden
+        schema:
+          $ref: "../swagger.yaml#/definitions/error"
+      500:
+        description: |-
+          Internal server error. Possible `error_code` attribute values:
+          * `internal_server_error`
+        schema:
+          $ref: "../swagger.yaml#/definitions/error"
+      503:
+        description: Under maintenance
+        schema:
+          $ref: "../swagger.yaml#/definitions/error"
+    x-koha-authorization:
+      permissions:
+        erm: 1
+  post:
+    x-mojo-to: ERM::CounterFiles#add
+    operationId: addErmCounterFiles
+    tags:
+      - counter_file
+    summary: Add counter_file
+    consumes:
+      - application/json
+    produces:
+      - application/json
+    parameters:
+      - description: A JSON object containing information about the new counter_file
+        in: body
+        name: body
+        required: true
+        schema:
+            $ref: "../swagger.yaml#/definitions/erm_counter_file"
+    responses:
+      201:
+        description: A successfully created counter_file
+        schema:
+          items:
+            $ref: "../swagger.yaml#/definitions/erm_counter_file"
+      400:
+        description: Bad parameter
+        schema:
+          $ref: "../swagger.yaml#/definitions/error"
+      401:
+        description: Authentication required
+        schema:
+          $ref: "../swagger.yaml#/definitions/error"
+      403:
+        description: Access forbidden
+        schema:
+          $ref: "../swagger.yaml#/definitions/error"
+      404:
+        description: Ressource not found
+        schema:
+          $ref: "../swagger.yaml#/definitions/error"
+      409:
+        description: Conflict in creating resource
+        schema:
+          $ref: "../swagger.yaml#/definitions/error"
+      413:
+        description: Payload too large
+        schema:
+          $ref: "../swagger.yaml#/definitions/error"
+      500:
+        description: |-
+          Internal server error. Possible `error_code` attribute values:
+          * `internal_server_error`
+        schema:
+          $ref: "../swagger.yaml#/definitions/error"
+      503:
+        description: Under maintenance
+        schema:
+          $ref: "../swagger.yaml#/definitions/error"
+    x-koha-authorization:
+      permissions:
+        erm: 1
+"/erm/counter_files/{erm_counter_files_id}":
+  get:
+    x-mojo-to: ERM::CounterFiles#get
+    operationId: getERMCounterFile
+    tags:
+      - counter_file
+    summary: get counter_file
+    produces:
+      - application/json
+    parameters:
+      - $ref: "../swagger.yaml#/parameters/erm_counter_files_id_pp"
+      - name: x-koha-embed
```

Comment:
What are we embedding here? Suggestion: Implement only the endpoints we need, instead of listing all to start with.